app/assets/stylesheets/static_pages.css.scss

```diff
@@ -0,0 +1,3 @@
+#welcome_message {
```

Comment:
Avoid using id selectors<br>Selector `welcome_message` should be written in lowercase with hyphens